python/core/auto_generated/qgspointlocator.sip.in

```diff
@@ -238,6 +256,35 @@ This method is either blocking or non blocking according to ``relaxed`` paramete
 .. versionadded:: 3.6
 %End
 
+    MatchList centroidsInRect( const QgsRectangle &rect, QgsPointLocator::MatchFilter *filter = 0 );
```

Comment:
```suggestion
    MatchList centroidsInRectangle( const QgsRectangle &rectangle, QgsPointLocator::MatchFilter *filter = 0 );
```

Reply:
(also with the other newly added methods)

Reply:
I agree with you. But was to match with older methods, and I think it's better to be consistent with the old ways, isn't it? Maybe a note fore QGIS 4?

Reply:
I'd say better do it now in this case, less refactoring is not bad :)

Reply:
Yes, but it's an API break?